compiler/quilt/test/test_import.py

```diff
@@ -3,6 +3,9 @@
 """
 
 import os
+import tempfile
+import time
+import quilt
```

Comment:
is this the way we're importing quilt?  or are we using a relative import to ensure we get the same version as the local git clone?

Reply:
Quilt base package can't be imported relatively from a module inside that package.  ..however, there's no need to use the base directly here.  File converted to use relative imports only.

There are a few other test files in general that could use that treatment, but that's probably a PR in itself, and not directly related.  Fixed since we touched this file.